internal/pkg/cli/pipeline_init.go

```diff
@@ -45,21 +45,26 @@ Please enter full repository URL, e.g. "https://github.com/myCompany/myRepo", or
 
 const (
 	buildspecTemplatePath = "cicd/buildspec.yml"
-	githubURL             = "github.com"
-	ccIdentifier          = "codecommit"
-	awsURL                = "aws.amazon.com"
-	ghProviderName        = "GitHub"
-	ccProviderName        = "CodeCommit"
-	defaultGHBranch       = "main"
-	defaultCCBranch       = "master"
-)
-
-const (
-	fmtSecretName     = "github-token-%s-%s"
+	// For a GitHub repository.
+	githubURL         = "github.com"
+	ghProviderName    = "GitHub"
+	defaultGHBranch   = "main"
 	fmtGHPipelineName = "pipeline-%s-%s-%s"
-	fmtCCPipelineName = "pipeline-%s-%s"
 	fmtGHRepoURL      = "https://%s/%s/%s"
+	fmtSecretName     = "github-token-%s-%s"
```

Comment:
I would suggest giving a comment (like `github-token-{appName}-{envName}`) for each of the format string like this otherwise it is pretty difficult to follow what to fill in `%s`...

Reply:
Done!